packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -394,6 +395,7 @@ class DatePicker extends React.Component {
   }
 
   handleOnRequestClose() {
+    this.setState({ isCalendarKeyboardFocused: false, isCalendarOpenedViaKeyboard: true });
```

Comment:
Should isCalendarOpenedViaKeyboard be set to false here? When you launch the picker using the keyboard, close it and the open it with a mouse click, the preselected day still has the dash border when it should not.

Reply:
Fixed 72febfc